### PR TITLE
Add Warning Message for Fillet Engine Limitations in CommandBar

### PR DESCRIPTION
**Pull request by max-mrgrsk**
this issue is part of Fillet project: #2606
closes: #3833

### Summary:
This PR adds a warning message to the CommandBar to inform users about current engine limitations with fillets (e.g., fillets cannot touch other fillets). The warning is visually distinct and aims to improve user awareness.

### Changes:
**Added `warningMessage`** to `CommandArgument` type and integrated it into the UI.

warningMessage

### Future Enhancements: 
The message will evolve to cover more fillet-related errors in the separate PRs, such as selecting already filleted edges, or engine fails on dry run : [#2612](https://github.com/KittyCAD/engine/issues/2612)

This is a first step toward improving user feedback for fillet operations.

### CODE REVIEW

File: src/lib/commandBarConfigs/settingsCommandConfig.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ const levelArgConfig = <T extends AnyStateMachine = AnyStateMachine>(
         ]
       : [{ name: 'User', value: 'user' as SettingsLevel, isCurrent: true }],
   machineActor: actor,
+  warningMessage: '',
```
Comment (franknoirot) [inline src/lib/commandBarConfigs/settingsCommandConfig.ts:52]: I think you could avoid the need for this if you make [this parameter optional](https://github.com/KittyCAD/modeling-app/pull/4076/files#diff-d718eecd2d14fae3f4ff21d3d9e0dd45af056ac3eb161f15a48d64c4183e5d75R193)

File: src/lib/commandTypes.ts [hunk 2]
```diff
@@ -189,6 +190,7 @@ export type CommandArgument<
       ) => boolean)
   skip?: boolean
   machineActor?: Actor<T>
+  warningMessage: string
```
Comment (franknoirot) [inline src/lib/commandTypes.ts:193]:
```suggestion
  warningMessage?: string
```

File: src/lib/kclCommands.ts [hunk 3]
```diff
@@ -126,6 +127,7 @@ export function kclCommands(
         sample: {
           inputType: 'options',
           required: true,
+          warningMessage: 'warning',
```
Comment (franknoirot) [inline src/lib/kclCommands.ts:130]: Similarly, I don't think you'll need these if you make that parameter optional.

**Review by franknoirot — COMMENTED** [review]
Nice, just a couple comments. I think it was a good move to make this a simple string for now, and not a full `ReactNode`.

I can see this parameter becoming more general in the future, so that instead of `warningMessage` it's something like `additionalInfo: { level: 'warning'; message: 'Fillets cannot touch other fillets yet. This is under development.' }`, so we can have other message levels like `info` and `destructive` that result in different-styled components. But we shouldn't do that until we need it.

**Comment by nadr0** [comment]
I agree with Frank's comments. I tested locally and it looks good. Glad it only shows up during the selection step.

File: src/lib/commandBarConfigs/settingsCommandConfig.ts (hunk 1 above)
Comment (max-mrgrsk) [reply src/lib/commandBarConfigs/settingsCommandConfig.ts]: good point, thanks!

**Review by franknoirot — APPROVED** [review]
Nice, that became a tiny PR!

**Pull request merged**